lib/http-bridge/test/unit/Cardano/Wallet/HttpBridge/BinarySpec.hs

```diff
@@ -143,6 +179,67 @@ spec = do
                 encodeSignedTx
                 (txs !! 0, [pkWit])
 
+    describe "decodeAddress <-> encodeAddress roundtrip" $ do
+        it "DerivationPath roundtrip" (property prop_derivationPathRoundTrip)
+
+    describe "Golden Tests for Byron Addresses w/ random scheme (Mainnet)" $ do
+        it "decodeDerivationPath - mainnet - initial account" $
+            decodeDerivationPathTest DecodeDerivationPath
+                { mnem =
+                    arbitraryMnemonic
+                , addr =
+                    "82d818584283581ca08bcb9e5e8cd30d5aea6d434c46abd8604fe4907d\
+                    \56b9730ca28ce5a101581e581c22e25f2464ec7295b556d86d0ec33bc1\
+                    \a681e7656da92dbc0582f5e4001a3abe2aa5"
```

Comment:
Note that these used to be base58-encoded for no particular reason. In the end, it was more confusing than anything else so I've re-encoded those as hexadecimal to not suggest that these are actual user-facing addresses.

Reply:
There _were_ actual user-facing address. I generated this test data with the old wallet API and copy&pasted directly into the tests.

Reply:
Of course, what I meant is that the fact that they were in base58 was unnecessary for the test and confused the reader into thinking that addresses had to be encoded in base58 to encode / decode the derivation path.
In this stage in testing, we only care about the binary payload of the address and not much about the user facing encoding. Hence why I switched from base58 to hexadecimal.